scripts/ci/jobs/diff-dumps.sh

```diff
@@ -92,9 +92,46 @@ upload_offline_dump() {
     if is_in_PR_context; then
         cmd+=(echo "Would do")
     fi
+    curl --silent --show-error --max-time 60 --retry 3 --create-dirs -o out/RELEASE_VERSION.txt https://raw.githubusercontent.com/stackrox/stackrox/master/scanner/updater/version/RELEASE_VERSION
+    version_file="out/RELEASE_VERSION.txt"
+    # Use grep to extract X.Y versions, sort them, and get the last one as the latest
+    latest_version=$(grep -oE '^[0-9]+\.[0-9]+' "$version_file" | sort -V | tail -n 1)
+
+    file_to_check="scanner-v4-defs-${latest_version}.zip"
+    
+    curl --silent --show-error --max-time 60 --retry 3 -o $file_to_check https://storage.googleapis.com/scanner-v4-test/offline-bundles/$file_to_check
+
+    if [ -f "$file_to_check" ]; then
+        # If the file exists, add it to scanner-vuln-updates.zip
```

Comment:
What is the scenario when the file does not exist? `curl` will fail if `offline-bundle/$file_to_check` returns a 404 NotFound; Since `set -e` the script we should never reach this line, right? Can you clarify?

Reply:
You are right. I should do
`If curl --silent --show-error --max-time 60 --retry 3 -o $file_to_check https://storage.googleapis.com/scanner-v4-test/offline-bundles/$file_to_check`